packages/@okta/vuepress-site/use_cases/inline_hooks/registration_hook/registration_hook/index.md

```diff
@@ -0,0 +1,244 @@
+
+| Property | Description                                           | Data Type       |
+|----------|-------------------------------------------------------|-----------------|
+| type     | One of the [supported commands](#supported-commands). | String          |
+| value    | Operand to pass to the command.                       | [value](#value) |
+
+For example commands, see the [value](#value) section below.
+
+#### Supported Commands
+
+The following commands are supported for the Registration Inline Hook type:
+
+| Command                      | Description                                                       |
+|------------------------------|-------------------------------------------------------------------|
+| com.okta.user.profile.update | Set or modify an attribute in the Okta user profile that will be created for this user.                 |
+| com.okta.action.update       | Allow or deny the user's registration. |
+
+To set attributes in the user's Okta profile, supply a type property set to `com.okta.user.profile.update`, together with a `value` property set to a list of key-value pairs corresponding to Okta user profile attributes you want to set. The attributes must already exist in your user profile schema.
+
+To explicitly allow or deny registration to the user, supply a type property set to `com.okta.action.update`, together with a value property set to `{"registration": "ALLOW"}` or `{"registration": "DENY"`}. The default is to allow registration.
+
+Commands are applied in the order in which they appear in the array. Within a single command, attributes are updated in the order in which they appear in the `value` object.
+
+You can never use a commands to update the user's password, but you are allowed to set the values of attributes other than password which are designated sensitive in your Okta user schema. Note however that the values of those sensistive attributes, if  included as fields in the Self-Service Registration form, are not included in the `data.userProfile` object sent to your external service by Okta. See [data.userProfile](#data-userProfile) above.
+
+#### value
+
+The `value` object is the parameter to pass to the command.
+
+For `com.okta.user.profile.update` commands, `value` should be an object containing one or more name-value pairs for the attributes you wish to update, for example:
+
+```json
+{
+   "commands":[
+      {
+         "type":"com.okta.user.profile.update",
+         "value":{
+            "middleName":"Danger",
+            "customerId":12345
+         }
+      }
+   ]
+}
+```
+
+> Note: The above example assumes that there is an attribute `customerId` defined in your Okta user schema (`middleName` is defined by default).
```

Comment:
Nit: I'm not sure this is important enough to be called out in its own "> Note:." I would probably just set it as normal body text.

Reply:
Removed note formatting.